pkg/helm/helm3lib/helm3lib.go

```diff
@@ -199,6 +200,13 @@ func (h *LibClient) upgradeRelease(releaseName string, chartName string, valuesP
 	upg.SkipCRDs = true
 	upg.MaxHistory = int(options.HistoryMax)
 	upg.Timeout = options.Timeout
+	// copy labels to avoid modifying the original map
+	if upg.Labels == nil {
+		upg.Labels = make(map[string]string)
+	}
+	if labels != nil {
+		maps.Copy(upg.Labels, labels)
```

Comment:
didn't get the clue. why not to pass labels directly to the upgrade cli. Do the upgradeCli modify them?

Reply:
also have to add to the installCli below